### PR TITLE
Generate mnemonic instead of seed

### PR DESCRIPTION
**Pull request by RafaelTaranto**

### CODE REVIEW

**Comment by RafaelTaranto** [comment]
[Switch seed file to mnemonic](https://trello.com/c/3w3vmSRD/707-switch-seed-file-to-mnemonic)

File: install [hunk 1]
```diff
@@ -12,8 +12,8 @@ CA_KEY_PATH=$KEY_DIR/Lamassu_OP_Root_CA.key
 CA_PATH=$CERT_DIR/Lamassu_OP_Root_CA.pem
 SERVER_KEY_PATH=$KEY_DIR/Lamassu_OP.key
 SERVER_CERT_PATH=$CERT_DIR/Lamassu_OP.pem
-SEEDS_DIR=$HOME/seeds
-SEED_FILE=$SEEDS_DIR/seed.txt
+MNEMONIC_DIR=$HOME/mnemonics
```
Comment (joshmh) [inline install]: Should be ``/etc/lamassu/mnemonics``

**Review by joshmh — CHANGES_REQUESTED** [review]

File: install (hunk 1 above)
Comment (RafaelTaranto) [reply install]: Even thought it was previously saving the seeds on $HOME/seeds?

Comment (joshmh) [reply install]: Hmm, I see. Yeah, I think /etc/lamassu is a better place for it and more consistent with where we're putting other stuff. Sorry about the confusion.

**Pull request merged**